modules/rosa-sts-required-roles-and-policies.adoc

```diff
@@ -0,0 +1,74 @@
+
+[id="rosa-sts-required-roles-and-policies-automatic_{context}"]
+== Automatic `ocm-role` resource creation
+
+You can use the automatic creation method in the CLI and from {cluster-manager} if you created an `ocm-role` resource with administrative permissions. Selecting this method creates the roles and policies that uses the default names.
+
+.Procedure
+
+* In the guided ROSA cluster installation in {cluster-manager}, select 'Auto mode' on step four, "Cluster roles and policies," to automatically create the roles and associate the policies by using the admin `ocm-role` resource.
++
+[NOTE]
+====
+If you use the ROSA guided installation on {cluster-manager}, you must have created an `ocm-role` resource with administrative permissions in the first step of the guided cluster installation. Without this role, you cannot use the automatic Operator role and policy creation option.
+====
+
+* In your preferred CLI, create the roles with the following command:
++
+[source,terminal]
+----
+rosa create account-role --mode auto
+----
++
+After creating these resources in your CLI or with the automatic processes, you have the following roles and policies:
++
+.Required operator roles
+[cols="1,2",options="header"]
+|===
+|Resource| Function on your cluster
+|`ManagedOpenShift-ControlPlane-Role`
+| An IAM role used by the ROSA control plane.
+|`ManagedOpenShift-Worker-Role`
+| An IAM role used by the ROSA compute instances.
+|`ManagedOpenShift-Support-Role`
+| An IAM role used by the Red Hat Site Reliability Engineering (SRE) support team.
+|`ManagedOpenShift-Installer-Role`
+| An IAM role used by the ROSA installer.
+|===
++
+.Required policies
+[cols="1,2",options="header"]
+|===
+|Resource| Function on your cluster
+|`ManagedOpenShift-openshift-cluster-csi-drivers-ebs-cloud-credent`
+| A managed IAM policy required by ROSA to manage back-end storage by using the Container Storage Interface (CSI).
+|`ManagedOpenShift-openshift-cloud-network-config-controller-cloud`
+| `NEED CONTENT`
+|`ManagedOpenShift-openshift-machine-api-aws-cloud-credentials`
+| A managed IAM policy that provides the ROSA Machine Config Operator with the permissions required to perform core cluster functions.
+|`ManagedOpenShift-openshift-cloud-credential-operator-cloud-crede`
+| A managed IAM policy that provides the ROSA Cloud Credential Operator with the permissions required to manage cloud provider credentials.
+|`ManagedOpenShift-openshift-image-registry-installer-cloud-creden`
+| A managed IAM policy that provides the ROSA Image Registry Operator with the permissions required to manage the internal registry storage in AWS S3 for a cluster.
+|`ManagedOpenShift-openshift-ingress-operator-cloud-credentials`
+| A managed IAM policy that provides the ROSA Ingress Operator with the permissions required to manage external access to a cluster.
+|===
+
+[id="rosa-sts-required-roles-and-policies-manual_{context}"]
+== Manual `ocm-role` resource creation
```

Comment:
The same comment applies to this heading. This heading suggests that the procedure is about creating the ocm-role resources, whereas it seems to be about creating the account-wide roles and inline policies, as well as the operator policies. You could update the heading to relate explicitly to the procedure.

Another point to note is that you can create these resources using auto mode or manual mode in the CLI, or using auto or manual mode in OCM. So, we need to distinguish between those paths too.